.eslintrc

```diff
@@ -0,0 +1,137 @@
+{
+  "parser": "babel-eslint",
+
+  "ecmaFeatures": {
+    "modules": true,
+    "experimentalObjectRestSpread": true
+  },
+
+  "env": {
+    "browser": false,
+    "es6": true,
+  },
+
+  "rules": {
+    // specify the maximum depth that blocks can be nested
+    "max-depth": [0, 4],
+    // specify the maximum length of a line in your program
+    "max-len": [0, 80, 4],
+    // limits the number of parameters that can be used in the function declaration.
+    "max-params": [0, 3],
+    // specify the maximum number of statement allowed in a function
+    "max-statements": [0, 10],
+    // disallow use of bitwise operators
+    "no-bitwise": 0,
+    // disallow use of unary operators, ++ and --
+    "no-plusplus": 0,
+
+    // enforce spacing inside array brackets
+    'array-bracket-spacing': 0,
+    // require camel case names
+    'camelcase': [2, {'properties': 'never'}],
```

Comment:
can properties be not camelcased? Which properties should never be camelcased? And what 2 is about, please?

```js
@props({
  urlRoot: 'comments'
})
```